bat/tests/06-test-format-bump/run.bats

```diff
@@ -7,45 +7,130 @@ setup() {
   global_setup
 }
 
-# TODO: This test should run 'mixer build upstream-format' instead
-# once the tooling required to run custom test docker images for
-# is ready.
-@test "Perform upstream format bump on last format boundary" {
+@test "Perform format bump using individual build format-bump <new/old>" {
   upstreamver=$(get-last-format-boundary)
   mixer-init-stripped-down $upstreamver 10
-
-  #Create initial version in the old format
-  mixer-build-all > $LOGDIR/build_all.log
-
-  #Save initial format
   format=$(sed -n 's/[ ]*FORMAT[ ="]*\([0-9]\+\)[ "]*/\1/p' mixer.state)
 
-  #Update to new upstream in the new format
-  mixer-upstream-update $(($upstreamver + 10)) > $LOGDIR/upstream_update.log
+  # create deprecated bundle
+  cat << EOF > local-bundles/editors
+# [TITLE]: editors
+# [DESCRIPTION]: Run popular terminal text editors
+# [STATUS]: Deprecated
+# [CAPABILITIES]:
+# [MAINTAINER]: Test Master <email@example.com>
+EOF
+  mixer bundle add editors
 
-  #Build +20
-  mixer-build-format-bump-new > $LOGDIR/build_formatbump_new.log
+  #Create initial version in the old format
+  mixer-build-bundles > $LOGDIR/build_bundles.log
 
-  #Reset upstream for +10 build
-  echo -n "$upstreamver" > upstreamversion
+  mixer-build-update > $LOGDIR/build_update.log
 
   #Build +10
-  mixer-build-format-bump-old > $LOGDIR/build_formatbump_old.log
+  format=$((format+1))
+  mixer-build-format-bump-old $format > $LOGDIR/build_formatbump_new.log
 
-  #restore upstrean
-  mv upstreamversion.bump upstreamversion
+  #Build +20
+  mixer-build-format-bump-new $format > $LOGDIR/build_formatbump_old.log
 
   #check if LAST_VER is set to +20
   test $(< update/image/LAST_VER) -eq 30
 
+  #check if deprecated bundle is marked deleted in +10
+  awk '
+  $4 == "editors" {
+    found = 1; exit $3 != 20
+  }
+  END {
+  if (found != 1)
+    exit 1
+  }' update/www/20/Manifest.MoM
+
+  #editors bundle should have all files marked as deleted in +10 (0's)
+  result="$(grep -E "(10|20)" update/www/20/Manifest.editors | grep -Ev "(version|previous|000000000)" | wc -l)"
+  [ "$result" -eq 0 ]
```

Comment:
Optional: if you want to use AWK instead, you could consolidate the above two lines into:

```
awk 'NF == 4 { if ($2 !~ /^0{64}$/) exit 1 }' update/www/20/Manifest.editors
```